src/views/StandaloneOfficeAddressFiling.vue

```diff
@@ -239,7 +239,7 @@ export default {
 
   data () {
     return {
-      updatedAddresses: {},
+      updatedAddresses: { registeredOffice: {}, recordsOffice: {} },
```

Comment:
More thorough init.
Probably not needed for prod code since this property gets assigned early on, but makes unit testing a bit nicer.